nrf802154_sniffer/nrf802154_sniffer.py

```diff
@@ -313,6 +313,9 @@ def control_reader(self, fifo):
                 arg, typ, payload = Nrf802154Sniffer.control_read(fn)
             self.stop_sig_handler()
 
+    def is_running(self):
```

Comment:
I don't see this function being used in this commit. Is it used by the application?